components/raftstore/src/coprocessor/split_check/size.rs

```diff
@@ -51,13 +51,6 @@ where
     E: KvEngine,
 {
     fn on_kv(&mut self, _: &mut ObserverContext<'_>, entry: &KeyEntry) -> bool {
-        // If there's no need to check region split, skip it.
```

Comment:
Why delete?

Reply:
because we want to leverage the scan (if it happened) to check if the region needs the split. So no need to skip it.

Reply:
The original bug that deleted change fixed no longer exists because the batch_split_limit won't be 0 anymore.